img2lambda/cli/main.go

```diff
@@ -15,6 +16,25 @@ import (
 	"github.com/urfave/cli"
 )
 
+// a list of aws supported runtimes as of 26/01/2019
+var validRuntimes = types.ValidRuntimes{
```

Comment:
This should be a const, and seems like a good thing to move entirely into the "types" file as a public value

Reply:
I have moved it to the types file but cannot declare it as a const as GoLang doesn't allow const slices. Let me know if you have a better approach in mind.